web/views/pages/index.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#" land="en">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#" lang="en">
```

Comment:
Good catch!

Reply:
I will attribute this one to [webhint](https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint) 😅